src/commands/community/nft_slash/config.ts

```diff
@@ -0,0 +1,77 @@
+import { CommandInteraction } from "discord.js"
+import { SlashCommandSubcommandBuilder } from "@discordjs/builders"
+import { CommandError, GuildIdNotFoundError } from "errors"
+import { composeEmbedMessage2 } from "utils/discordEmbed"
+import { SLASH_PREFIX } from "utils/constants"
+import { SlashCommand } from "types/common"
+import { handle } from "../nft/config"
+
+const command: SlashCommand = {
+  name: "config_twitter-sale",
+  category: "Community",
+  prepare: () => {
+    return new SlashCommandSubcommandBuilder()
+      .setName("config_twitter-sale")
+      .setDescription("Config twitter sales bot")
+      .addStringOption((option) =>
+        option
+          .setName("consumer_key")
+          .setDescription("Your twitter consumer key. Example: J9ts...")
+          .setRequired(true)
+      )
+      .addStringOption((option) =>
+        option
+          .setName("consumer_key_secret")
+          .setDescription("Your twitter consumer key secret. Example: hNl8...")
+          .setRequired(true)
+      )
+      .addStringOption((option) =>
+        option
+          .setName("access_token")
+          .setDescription("Your twitter access token. Example: 1450...")
+          .setRequired(true)
+      )
+      .addStringOption((option) =>
+        option
+          .setName("access_token_secret")
+          .setDescription("Your twitter access token secret. Example: POvv...")
+          .setRequired(true)
+      )
+  },
+  run: async function (interaction: CommandInteraction) {
+    if (!interaction.guild || !interaction.guildId) {
+      throw new GuildIdNotFoundError({})
+    }
+
+    const csmrKey = interaction.options.getString("consumer_key")
+    const csmrKeyScrt = interaction.options.getString("consumer_key_secret")
+    const acsToken = interaction.options.getString("access_token")
+    const acsTokenScrt = interaction.options.getString("access_token_secret")
```

Comment:
edit to match above and also define the string values upfront at the start of the file and use them below
e.g
```typescript
import .... from ...
import .... from ...

const CONSUMER_KEY = "consumer key"
const CONSUMER_KEY_SECRET = "consumer key secret"
...

// use them below

```